app/Services/Three/SignupService.php

```diff
@@ -56,6 +57,50 @@ public function create($data, $northstarId)
         return $signup;
     }
 
+    /**
+     * Handles all business logic around updating signups.
+     *
+     * @param array $data
+     * @param string $northstarId
+     * @return Illuminate\Database\Eloquent\Model $model
+     */
+    public function update($signup, $data)
```

Comment:
Need to update the comment block to match the actual params

Reply:
@sbsmith86 thank you for catching all of these!